backend/Api/Forecasts/ReadModelFactory.cs

```diff
@@ -0,0 +1,210 @@
+
+        var totalOffered = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Offer);
+
+        var totalAbsence = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.PlannedAbsence);
+
+        var totalNotStartedOrQuit = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.NotStartedOrQuit);
+
+        var totalExcludableAbsence = detailedBookingsArray
+            .Where(s => s.BookingDetails is { Type: BookingType.PlannedAbsence, ExcludeFromBilling: true })
+            .Select(wh => wh.TotalHoursForMonth(month))
+            .Sum();
+
+        var totalVacations = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Vacation);
+
+        var bookedTime = totalBillable + totalAbsence + totalVacations + totalHolidayHours + totalNonBillable + totalNotStartedOrQuit;
+        var forecastedBookedTime = bookedTime + totalOffered;
+
+        var workHoursInMonth = WorkloadHelper.CalculateWorkHoursInMonth(month, organization);
+
+        var totalSellableTime = Math.Max(workHoursInMonth - forecastedBookedTime, 0);
+        var totalOverbooked = Math.Max(bookedTime - workHoursInMonth, 0);
+
+        return new BookedHoursInMonth(
+            month,
+            new BookingReadModel(
+	            totalBillable,
+	            totalOffered,
+	            totalAbsence,
+	            totalExcludableAbsence,
+                totalSellableTime,
+                totalHolidayHours,
+	            totalVacations,
+                totalOverbooked,
+	            totalNotStartedOrQuit)
+        );
+	}
+
+	private static IEnumerable<ForecastForMonth> GetForecasts(Consultant consultant, List<DateOnly> months, List<BookedHoursInMonth> bookingSummary)
+	{
+		foreach (var month in months)
+		{
+			var forecastPercentage = consultant.Forecasts
+				.SingleOrDefault(f => f.Month.EqualsMonth(month))?
+				.AdjustedValue ?? 0;
+
+			var booking = bookingSummary.SingleOrDefault(bs => bs.Month.EqualsMonth(month))?.BookingModel;
+
+			if (booking == null)
+			{
+				yield return new ForecastForMonth(month, 0, forecastPercentage);
+				continue;
+			}
+
+			var bookedPercentage = CalculateBookedPercentage(month, consultant, booking);
+
+			var displayedPercentage = Math.Max((int)bookedPercentage, forecastPercentage);
+
+			yield return new ForecastForMonth(month, bookedPercentage, displayedPercentage);
+		}
+	}
+
+	private static double CalculateBookedPercentage(DateOnly month, Consultant consultant, BookingReadModel booking)
+	{
+		var workHoursInMonth = WorkloadHelper.CalculateWorkHoursInMonth(month, consultant.Department.Organization);
+
+		var forecastedBookedHours = workHoursInMonth - booking.TotalSellableTime;
+
+		return 100 * (forecastedBookedHours / workHoursInMonth);
```

Comment:
```suggestion
		return 100 * (booking.TotalBillable / workHoursInMonth);
```

Scratch `forecastedBookedHours` and simply use `booking.TotalBillable` here

Reply:
Fixed in https://github.com/varianter/vibes/pull/605/commits/21bfa1f2f8241d97dbe8da5342da6023d2c0bbe4